docs/develop/python/message-passing.mdx

```diff
@@ -244,73 +261,94 @@ async def main():
     )
 ```
 
-### Sending an Update {#send-update}
+### Send an Update {#send-update-from-client}
 
-- A client sending an Update must be prepared to wait until the Server has delivered the Update to the Worker: that means that the Worker must not be offline, and must not respond too slowly.
-  If you want the server to send a response as soon as it receives your request, then you must use a Signal instead.
-- `WorkflowExecutionUpdateAccepted` will be added to Event History when the Worker has responded to the Server that the Update passed validation.
-- `WorkflowExecutionUpdateCompleted` will be added to Event History when the Worker has responded to the Server that the Update has completed.
+An Update is a synchronous, blocking call that can change Workflow state, control its flow, and return a result.
 
+A client sending an Update must wait until the Server delivers the Update to a Worker.
+Workers must be available and responsive.
+If you need a response as soon as the Server receives the request, use a Signal instead.
+Also note that you can't send Updates to other Workflow Executions or perform an Update equivalent of Signal-With-Start.
 
-It is not yet possible to send an Update to another Workflow, or to do the Update equivalent of Signal-With-Start.
+- `WorkflowExecutionUpdateAccepted` is added to the Event History when the Worker confirms that the Update passed validation.
+- `WorkflowExecutionUpdateCompleted` is added to the Event History when the Worker confirms that the Update has finished.
 
-To send an Update to a Workflow Execution, you have two choices:
+To send an Update to a Workflow Execution, you can:
 
-#### 1. Use [`execute_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#execute_update) to wait for the update to complete
+- Call [`execute_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#execute_update) and wait for a Worker to complete.
+  This code fetches an Update result:
 
-[`execute_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#execute_update) sends an Update and waits until it has been completed by a Worker. It returns the Update result:
+  ```python
+  previous_language = await workflow_handle.execute_update(
+      GreetingWorkflow.set_language, Language.Chinese
+  )
+  ```
 
-```python
-previous_language = await workflow_handle.execute_update(
-    GreetingWorkflow.set_language, Language.Chinese
-)
-```
-
-#### 2. Use [`start_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#start_update) to receive a handle as soon as the update is accepted or rejected
+- Send [`start_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#start_update) to receive an [`UpdateHandle`](https://python.temporal.io/temporalio.client.WorkflowUpdateHandle.html) as soon as the update is accepted or rejected.
```

Comment:
```suggestion
- Send [`start_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#start_update) to receive an [`UpdateHandle`](https://python.temporal.io/temporalio.client.WorkflowUpdateHandle.html) as soon as the Update is accepted or rejected.
```

Reply:
Document audited now for Update, Activity, Signal, Query, Workflow, Worker -- all outside code. Your comments are untouched.